kgfx.cpp

```diff
@@ -151,17 +148,17 @@ void KGFX::drawChart(std::vector<float> arr, int color, int y, int spacing, int
   chartSpr.createPalette(palette);
 
   int *fa = fmtChartArray(arr, height);
-  for (int i=0;i<(chartLen-2);i++) {
-    drawGraphLine(i*spacing, fa[i+1], (i+1)*spacing, fa[i+2], color);
+  for (int i = 0; i < (chartLen - 2); i++) {
+    drawGraphLine(i * spacing, fa[i + 1], (i + 1) * spacing, fa[i + 2], color);
   }
 
   int multi = 5;
-  if (height>80) {
-    multi = height/13;
+  if (height > 80) {
+    multi = height / 13;
   }
-  for (int i=0;i<240;i++) {
-    for (int j=0;j<height;j++) {
-      if (TFT_BLACK == chartSpr.readPixel(i, j) && (j>0 && TFT_BLACK != chartSpr.readPixel(i, j-1))) {
+  for (int i = 0; i < 240; i++) {
+    for (int j = 0; j < height; j++) {
```

Comment:
Hmm, are these spaces a personal preference or enforced by your editor?

Reply:
They must be. The linter that this editor has enforces some icky rules. I'll fix them.

Reply:
fixed.